.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Before filling out this form,
+        please review the (Cluster and OS Pre-Requisites)[https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md],
+        as well as the (BYOH Instance Pre-requisites)[BYOH Instance Pre-requisites].
+  - type: checkboxes
+    id: community-check
+    attributes:
+      label: Community Issue
+      description: |
+        Github issues are for those using the community distribution of WMCO.
+        Support for users with a subscription should be requested through the Red Hat Customer Portal.
+      options:
+        - label: I am using the community version of WMCO
+          required: true
+  - type: dropdown
+    id: cluster-type
+    attributes:
+      label: Version
+      description: Are you using an OKD or OCP cluster?
+      options:
+        - OKD
+        - OCP
+    validations:
+      required: true
+  - type: input
+    id: cluster-version
+    attributes:
+      label: Cluster Version
+      description: What is the version of your cluster?
+      placeholder: 4.12
+    validations:
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: What platform is your cluster running on?
+      options:
+        - AWS
+        - Azure
+        - Bare metal
+        - GCP
```

Comment:
GCP?

Reply:
Thats true, I'd prefer to leave the option so that if someone is extremely intent on filing a bug for an unsupported platform they still can. Its possible they will select any other platform as the field is required and that will make our lives harder.
I've also added `bare metal` as it is another very popular platform type, and an `other` field just in case.

I've already linked the cluster pre-requisites which clearly state which is supported and which isnt

Reply:
I suggest leaving it out. We should only list supported platforms here.